mass_mailing_list_dynamic/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,5 @@
+* `Tecnativa <https://www.tecnativa.com>`_:
```

Comment:
I think one of the things we want is to remove the email addresses here to prevent users from writing to these emails for support. Am I right, @pedrobaeza ?